nixos/modules/services/misc/persistent-evdev.nix

```diff
@@ -0,0 +1,64 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
```

Comment:
Please try to avoid this: https://nix.dev/anti-patterns/language#with-attrset-expression